src/brogue/RogueMain.c

```diff
@@ -1110,6 +1113,10 @@ void gameOver(char *killedBy, boolean useCustomPhrasing) {
         saveRecording(recordingFilename);
     }
 
+    if (rogue.playbackMode && nonInteractivePlayback) {
+        printf("Recording: %s ended after %li turns (game over).\n", rogue.currentGamePath, rogue.playerTurnNumber);
```

Comment:
What if we also include how the player died, the depth level, and the amount of gold? Could be useful for collecting stats from recordings.

Reply:
I think that could be a follow-up enhancement, if we find that use case useful.